Formula/stout.rb

```diff
@@ -0,0 +1,22 @@
+require "language/go"
+
+class Stout < Formula
+  desc "Reliable static website deploy tool"
+  homepage "http://stout.is"
+  url "https://github.com/EagerIO/Stout/archive/v1.2.2.tar.gz"
+  sha256 "e04a44735c4c04ccce63e73d211d2a8c0232594b23c8dfc5c971c10d1aa72aaa"
+  depends_on "go" => :build
+  depends_on "godep" => :build
```

Comment:
If you're using the `vendor` system upstream we're more than happy to use the vendor system here. Our preference is more or less:

Vendor => Godeps => `go_resource`s, where all three are an option.